openerp/http.py

```diff
@@ -315,7 +315,10 @@ def checked_call(___dbname, *a, **kw):
     def debug(self):
         """ Indicates whether the current request is in "debug" mode
         """
-        return 'debug' in self.httprequest.args
+        debug = 'debug' in self.httprequest.args
+        if not debug and self.httprequest.referrer:
```

Comment:
What is the purpose of this change ? I mean, the use case that triggers this case ?